sql/core/src/main/java/org/apache/spark/sql/sources/v2/reader/streaming/StreamingReadSupport.java

```diff
@@ -20,11 +20,10 @@
 import org.apache.spark.sql.sources.v2.reader.ReadSupport;
 
 /**
- * A base interface for streaming read support. This is package private and is invisible to data
- * sources. Data sources should implement concrete streaming read support interfaces:
- * {@link MicroBatchReadSupport} or {@link ContinuousReadSupport}.
+ * A base interface for streaming read support. Data sources should implement concrete streaming
+ * read support interfaces: {@link MicroBatchReadSupport} or {@link ContinuousReadSupport}.
```

Comment:
Hm .. would it be better to leave a comment like "This is exposed for a testing purpose." if this PR targets to expose this only for the testing purpose?

Reply:
Sure, I can add that. For now, I just used the way of `public interface ReadSupport`. I'll update it later.

Reply:
BTW, this doesn't fail in `sbt` and `IntelliJ` for now. I updated the PR description.

Reply:
I added the comment, @HyukjinKwon .